R/utilities-units.R

```diff
@@ -495,7 +496,8 @@ initializeDimensionAndUnitLists <- function() {
 
   # `yErrorUnit` column won't be present when only simulated datasets are
   # entered, but it can be assumed to be the same as `yUnit`.
-  if (!"yErrorUnit" %in% names(data)) {
+  if (("yErrorValues" %in% names(data)) &&
```

Comment:
Could you explain this change?

Reply:
If there is no `yErrorValues` column in the entered data frame, it doesn't make sense for this function to introduce a new column called `yErrorUnit`.